include/gatekeeper_net.h

```diff
@@ -262,6 +276,27 @@ int gatekeeper_get_rss_config(uint8_t portid,
 int gatekeeper_init_network(struct net_config *net_conf);
 void gatekeeper_free_network(void);
 
+static inline bool
+ipv4_if_configured(struct gatekeeper_if *iface)
+{
+	return !!(iface->configured_proto & GK_CONFIGURED_IPV4);
+}
+
+static inline bool
+ipv6_if_configured(struct gatekeeper_if *iface)
+{
+	return !!(iface->configured_proto & GK_CONFIGURED_IPV6);
+}
+
+static inline int
+max_prefix_len(int ip_type)
+{
+	RTE_VERIFY(ip_type == AF_INET || ip_type == AF_INET6);
+	return 8 * (ip_type == AF_INET
+		? sizeof(struct in_addr)
+		: sizeof(struct in6_addr));
```

Comment:
It would be better to rewrite the `return` as follows:
```
return ip_type == AF_INET
	? sizeof(struct in_addr) * 8
	: sizeof(struct in6_addr) * 8;
```
The motivation for this change is that the compiler will see a product of two constants and use just the final product. In the original form of the code, the compiler may not find this tiny optimization.